frontend/src/components/organisms/MatchesList/MatchesList.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {useEffect} from 'react';
+import styled from "styled-components";
+import {Card} from '@material-ui/core';
+import {MatchItem} from "../../molecules/MatchItem/MatchItem";
+import {MIN_CARD_COMPONENT_WIDTH} from "../../atoms/constants/sizes";
+import {MatchListItem} from "./MatchListItem";
+import {MatchesListRestApi} from "../../../restapi/matches-list";
+import {MatchesListDto} from "./MatchesListDto";
+import {MatchStatus} from "./MatchStatus";
+
+const StyledMatchesList = styled(Card)({
+    width: MIN_CARD_COMPONENT_WIDTH,
+});
+
+export type MatchesListProps = {
+    readonly tournamentId: string;
+};
+
+export const MatchesList = ({tournamentId}: MatchesListProps) => {
+
+    const [expanded, setExpanded] = React.useState<string | boolean>(false);
```

Comment:
Jak to jest, że string | boolean :O ?

Reply:
No bo ta ich stronie  dawali w ten sposób ale poprawię, bo chyba idzie to naprawić na pustego stringa!

Reply:
Chyba wiekszy sens ma boolean :) No bo expandend true/false?

Reply:
Expand otrzymuje np> false,
 "panel1", "panel2", panel3"....

Reply:
To moze musi tak zostac. A Ty gdzies odczytujesz ktory panel jest otwarty :) ?

Reply:
Mam dwa komponenty 
W tym drugim i przekazuję mu info

![image](https://user-images.githubusercontent.com/41823689/114305133-2be4e780-9ad7-11eb-8254-e5ebc0a6fe79.png)

Reply:
Tutaj w match Item

![image](https://user-images.githubusercontent.com/41823689/114305173-4d45d380-9ad7-11eb-826e-dee56f2bfac8.png)